models/cooperation.js

```diff
@@ -0,0 +1,43 @@
+const { Schema, model } = require('mongoose')
+const {
+  enums: { COOPERATION_STATUS }
+} = require('~/consts/validation')
+
+const cooperationSchema = new Schema(
+  {
+    offerId: {
+      type: Schema.Types.ObjectId,
+      ref: 'Offer',
+      required: [true, 'Offer ID must be valid.']
+    },
+    initiatorUserId: {
+      type: Schema.Types.ObjectId,
+      ref: 'User',
+      required: [true, 'ID of initiator user must be valid.']
```

Comment:
'user' is redundant here
me thinks
also this option is not about being valid, but about being required
change it to 'Please select blahblah' or 'this field cannot be empty' if anything
better this field cannot be empty tho